test/ipaddress/ipv4_test.rb

```diff
@@ -491,6 +491,16 @@ def test_method_subnet
     arr = ["172.16.10.0/24"]
     assert_equal arr, @network.subnet(24).map {|s| s.to_string}
   end
+
+  def test_method_subtract
```

Comment:
I find this test fairly hard to read and understand. Variable naming and maybe more than 1 test would make it better.

Reply:
I will use the same convention as for the sample.